pkg/inngest/inngest/_internal/server_lib/registration.py

```diff
@@ -157,6 +158,52 @@ def serialize_period(
         return out
 
 
+class Timeouts(_BaseConfig):
+    start: typing.Union[int, datetime.timedelta, None] = None
+    finish: typing.Union[int, datetime.timedelta, None] = None
+
+    @pydantic.field_serializer("start")
+    def serialize_start(
+        self,
+        value: typing.Union[int, datetime.timedelta, None],
+    ) -> typing.Optional[str]:
+        if value is None:
+            return None
+        out = transforms.to_duration_str(value)
+        if isinstance(out, Exception):
+            raise out
+        return out
+
+    @pydantic.field_serializer("finish")
+    def serialize_finish(
+        self,
+        value: typing.Union[int, datetime.timedelta, None],
+    ) -> typing.Optional[str]:
+        if value is None:
+            return None
+        out = transforms.to_duration_str(value)
+        if isinstance(out, Exception):
+            raise out
+        return out
+
+
+class MyModel(pydantic.BaseModel):
+    """
+    User model.
+
+    Attributes:
+        name: The user's full name
+        age: Age in years, must be positive
+    """
+
+    name: str = pydantic.Field(description="The user's full name")
+    age: int = pydantic.Field(description="Age in years, must be positive")
+
+
+bar = MyModel(name="foo", age=1)
+bar.name
+
+
```

Comment:
this looks like some kind of leftover test code?

Reply:
Oops! Yea, this was me testing something in Pydantic